src/main/assets/scss/_ie8.scss

```diff
@@ -0,0 +1,10 @@
+// Replace deprecated @govuk/frontend mixins to maintain compatibility with @hmcts/frontend
+// We will look to remove @hmcts/frontend and these mixins in NFDIV-3967
+$govuk-is-ie8: false;
+$govuk-ie8-breakpoint: ('desktop');
+@mixin govuk-if-ie8 {
+  @if false {
+    @content;
+  }
+}
+@mixin govuk-not-ie8 {@content}
```

Comment:
These mixins can be removed when we swap out @hmcts/frontend components and delete the dependency in NFDIV-3967. I thought it was best to handle that in a separate ticket so the govuk-frontend migration steps in this PR can be tested independently and there's less risk of bugs